packages/shared/src/components/auth/OnboardingAnalytics.tsx

```diff
@@ -5,6 +5,27 @@ import { isProduction } from '../../lib/constants';
 export const FB_PIXEL_ID = '519268979315924';
 export const GA_TRACKING_ID = 'AW-619408403';
 
+export const HotJarTracking = (): ReactElement => {
+  if (!isProduction) {
+    return null;
+  }
+
+  return (
+    <Script strategy="afterInteractive" id="load-hotjar">
+      {`
+      (function(h,o,t,j,a,r){
+        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+        h._hjSettings={hjid:3871311,hjsv:6};
+        a=o.getElementsByTagName('head')[0];
+        r=o.createElement('script');r.async=1;
+        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+        a.appendChild(r);
+      })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
```

Comment:
The above might block some rendering due to script just injecting another `<script />` tag into `head` thus escaping Next.js Script mechanism. But let's add like this and then if it makes issues we can refactor so that `static.hotjar.com` and inline code is executed by `Script`.

Reply:
@capJavert we gonna keep it there only for a few days